Copilot/README.md

```diff
@@ -0,0 +1,350 @@
+2. A Copilot license enabled for at least one user in that tenant. You have two ways to get a developer environment for Copilot:
+    - [A sandbox Microsoft 365 tenant](https://developer.microsoft.com/en-us/microsoft-365/tap) with Copilot (available in limited preview through TAP membership). 
+    - [An eligible Microsoft 365 or Office 365 production environment](https://learn.microsoft.com/en-us/microsoft-365-copilot/extensibility/prerequisites#customers-with-existing-microsoft-365-and-copilot-licenses) with a Copilot for Microsoft 365 license.
+
+3. A working React SharePoint Embedded application, written in TypeScript.
+    - If you are not using TypeScript, please see the Appendix below for steps to take to get this running in your app.
+    - To get started with SharePoint Embedded, visit https://aka.ms/start-spe
+    - Then, if you want get started with a sample React SPE app, visit [spe-azurefunction](https://github.com/microsoft/SharePoint-Embedded-Samples/tree/main/Samples/spa-azurefunction)
+4. Your Container Type configuration for `DiscoverabilityDisabled` must be set to `false`
+    - Visit [Configuring Container Types](https://learn.microsoft.com/en-us/sharepoint/dev/embedded/concepts/app-concepts/containertypes#configuring-container-types) to learn how to set this configuration
+
+## Quick Start
+
+### 1. Use the `SharePoint-Embedded-Samples\Samples\spe-typescript-react-azurefunction` Application
+
+Naviagate to the `SharePoint-Embedded-Samples\Samples\spe-typescript-react-azurefunction\react-client\src\components\ChatSidebar.tsx` file and adjust the Function Component by replacing the return statement with the code that is commented out in the file. It should look somewhat like this:
+```typescript
+export const ChatSidebar: React.FunctionComponent = () => {
+
+    const [chatAuthProvider, setChatAuthProvider] = React.useState<ChatAuthProvider | undefined>();
+    
+    const [chatConfig] = React.useState<ChatLaunchConfig>({
+        header: ChatController.instance.header,
+        theme: ChatController.instance.theme,
+        zeroQueryPrompts: ChatController.instance.zeroQueryPrompts,
+        suggestedPrompts: ChatController.instance.suggestedPrompts,
+        instruction: ChatController.instance.pirateMetaPrompt,
+    });
+
+   ...
+
+   return (<>
+    {chatAuthProvider && (
+        <ChatEmbedded
+            authProvider={chatAuthProvider}
+            onApiReady={onApiReady}
+        />
+    )}
+    </>);   
+}
+```
+
+Then navigate to the `Q:\src\SPE-Demo\SharePoint-Embedded-Samples\Samples\spe-typescript-react-azurefunction\react-client\src\routes\App.tsx` file and look for this line:
```

Comment:
This assumes that `Q:\src\SPE-Demo\` is a location that exists in all machines. Is this use of `Q:\` intentional?